src/pages/UserPage/UserPage.tsx

```diff
@@ -34,12 +133,15 @@ const UserPage: FC = () => {
   }
 
   const userNameString = `${user.user.forename} ${user.user.surname}`
+  const isUserDeactivated = !!deactivationDate
+
+  const deactivatedDate = dayjs(deactivationDate, 'YYYY-MM-DD').toDate()
+  const isDeactivationDatePastCurrentDate =
```

Comment:
**[non-blocker]**

Would be better to use dayjs whereever possible and basically use Date only when there are no other options.
Currently dayjs is used, but not for comparing 2 dates, which is also a feature that is build into dayjs.

So if you want to check whether 1 date is before or after another, you can use dayjs objects instead and use the isBefore or isAfter method of dayjs.

Could reduce the code a bit + avoid unnecessary conversions to Date object.

https://day.js.org/docs/en/query/is-after
https://day.js.org/docs/en/query/is-before#docsNav